client/friends/friends.controller.js

```diff
@@ -0,0 +1,21 @@
+(function() {
+    'use strict';
+    angular.module('closeness')
+        .controller('friendCtrl', friendCtrl);
+
+    function friendCtrl(facebookFriend) {
+        console.log('Friends invite Controller');
+        facebookFriend.getFriends().then(function(data) {
+            this.friendsList = data.data.data;
+        }, function(data) {
+            console.log('Unable to Get Friend List');
+        });
+        this.selected = null;
+        this.friendClick = function(index) {
+            this.selected = index;
+        }
+        this.invite = function() {
+            window.plugins.socialsharing.share('Message only');
```

Comment:
can u make a factory for social sharing and there check if window.plugins is defined or not. because this will give error in browser